trillian/ctfe/instance.go

```diff
@@ -222,6 +225,20 @@ func ValidateLogMultiConfig(cfg *configpb.LogMultiConfig) (map[string]*configpb.
 		if ok := logIDMap[logIDKey]; ok {
 			return nil, fmt.Errorf("log config: dup tree id: %d for: %v", logCfg.LogId, logCfg)
 		}
+		if start, limit := logCfg.GetNotAfterStart(), logCfg.GetNotAfterLimit(); start != nil || limit != nil {
```

Comment:
This feels like it would flow better as separate chunks for start then limit, then check the pair after.
```go
var tStart time.Time
start := logCfg.GetNotAfterStart()
if start != nil {
	tStart, err = ptypes.Timestamp(start)
	if err != nil {
		return nil, fmt.Errorf("log_config: invalid start timestamp %v for: %v", err, logCfg)
	}
}
var tLimit time.Time
limit := logCfg.GetNotAfterLimit()
if limit != nil {
	tLimit, err = ptypes.Timestamp(limit)
	if err != nil {
		return nil, fmt.Errorf("log_config: invalid limit timestamp %v for: %v", err, logCfg)
	}
}
if start != nil && limit != nil && tLimit.Before(tStart) {
￼ 	return nil, fmt.Errorf("log_config: limit before start for: %v", logCfg)
￼ }
```

Reply:
Is that better? Both options look somewhat ugly. I almost might write a helper that accepts nil.

Reply:
OK done but I don't think the vars are less ugly.

Reply:
Hmm, looks like GitHub lost a comment of mine where I said I didn't mind either way...

Reply:
Yeah, doesn't seem like much in it. I also tried the helper approach and ironically it didn't really help.